index.html

```diff
@@ -673,17 +673,22 @@ <h2>Expiration</h2>
 
       <section>
         <h2>Status</h2>
-        <p>Credential status information, such as suspension or revocation,
-        may be provided by adding the following <a>property</a>:
+        <p>Information about the current status of a <a>verifiable credential</a>, 
+        such as suspension or revocation,
+        may be provided by adding the credentialStatus <a>property</a>. This property
```

Comment:
nit: credentialStatus needs to be wrapped in code tags: i.e. <code>credentialStatus<code>

Reply:
done